doc/man/bitcoin-qt.1

```diff
@@ -317,10 +317,6 @@ created within past week. 0 = no limit (default: 0M). Optional
 suffix units [k|K|m|M|g|G|t|T] (default: M). Lowercase is 1000
 base while uppercase is 1024 base
 .HP
-\fB\-natpmp\fR
```

Comment:
In 1b1f359fc43385cb4d465b15754dac84eef06873 "[doc] manpages for 28.3rc1"

I don't think this is supposed to be removed from the manage.

Same with `-upnp`.

Reply:
Is there a configure flag I need? I'm just running gen-manpages.py after building.